gatling/user-files/simulations/HttpReplaceDocument.scala

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2011-2018 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package computerdatabase
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import scala.concurrent.duration._
+import sys.process._
+
+class HttpReplaceDocument extends Simulation {
+  val host = System.getProperty("host", "localhost")
+  val requests = System.getProperty("requests", "2000").toInt
+  val users = System.getProperty("users", "1").toInt
+  val duration = System.getProperty("duration", "1").toInt
+  var jwt = System.getProperty("jwt", "some jwt")
+  
+
+  val document = """
+      {
+        "driver": {
+          "name": "Eltooooon",
+          "age": 42,
+          "license": "B"
+        },
+
+        "car": {
+          "position": {
+            "lat": 42.83734827,
+            "lng": 8.298382039
+          },
+          "type": "berline"
+        }
+      }
+    """
+
+  val result = Process("""node ./user-files/utils/requestOneId""")
+  val exitCode = result.!
+  val input_file = "./id.txt"
+  val id = scala.io.Source.fromFile(input_file).mkString
+
+  val httpProtocol = http
+    .baseUrl("http://" + host + ":7512")
+    .acceptHeader("text/html,application/json,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8")
+    .acceptEncodingHeader("gzip, deflate")
+    .userAgentHeader("Gatling2")
+
+  val scn = scenario("Http replace document")
+    .repeat(requests, "i") {
+    exec(http("document:replace")
+        .put("http://" + host + ":7512/nyc-open-data/yellow-taxi/"+ id +"/_replace")
```

Comment:
nitpicking: you can use a s interpolator:

```suggestion
        .put(s"http://${host}:7512/nyc-open-data/yellow-taxi/${id}/_replace")
```